stdlib/CMakeLists.txt

```diff
@@ -79,6 +86,10 @@ option(SWIFT_STDLIB_STABLE_ABI
        "Should stdlib be built with stable ABI (library evolution, resilience)."
        "${SWIFT_STDLIB_STABLE_ABI_default}")
 
+option(SWIFT_STDLIB_OBJC_INTEROP
+       "Should stdlib be built with Obj-C interop."
+       "${SWIFT_STDLIB_OBJC_INTEROP_default}")
```

Comment:
Oh, this is a good idea!  I didn't even think of this.  What do you think of `SWIFT_STDLIB_ENABLE_OBJC_INTEROP` instead?